ingestion/tests/unit/test_suite/test_validations_datalake.py

```diff
@@ -183,7 +183,7 @@
             "test_case_column_values_to_be_unique",
             "columnValuesToBeUnique",
             "COLUMN",
-            (TestCaseResult, "20", "2", TestCaseStatus.Failed),
+            (TestCaseResult, "20", "0", TestCaseStatus.Failed),
```

Comment:
Should this be 1? This nickname `NickName` should be unique right?

Reply:
I went through the test case data..
the unique value were 0 because
it was repeating..